akka-actor-typed-tests/src/test/resources/logback-doc-dev.xml

```diff
@@ -6,14 +6,14 @@
             <level>INFO</level>
         </filter>
         <encoder>
-            <pattern>[%date{ISO8601}] [%level] [%logger] [%marker] [%thread] - %msg {%mdc}%n</pattern>
+            <pattern>[%date{ISO8601}] [%level] [%logger] [%marker] [%thread] - %msg MDC: {%mdc}%n</pattern>
```

Comment:
I'm open to other suggestions. 
```
MDC: [%mdc]
```

the reason for using `{ }` was that it's like json (but not full syntax)

Reply:
I personally prefer `[]`, but fine by me.